src/WordSearchToDataBase.java

```diff
@@ -0,0 +1,87 @@
+import java.sql.*;
+import java.text.MessageFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+
+/**
+ * Creating the connection by using jdbc.
+ * Checking if table ia already present or not, if present we are inserting the values.
+ * If table not present Creating a table with some columns which is the data.
+ * Calling method createNewTableAndInsertValues to create a new table and insert the values in it.
+ */
+class WordSearchToDataBase {
+    String driverClass = "com.mysql.cj.jdbc.Driver";
+    String sqlUrl = "jdbc:mysql://localhost:3306/WordSearchPoc1";
+    String userName = "root";
+    String dbPassword = "Armyairforce@1";
+    String dateAndTimeFormat = "dd/MM/yyyy HH:mm:ss";
+    String createTable = "create table audit (PathToTheFile varchar(100), SearchedWord varchar(45), DateAndTimeOfSearch varchar(45), Result varchar(45), WordCount int, ErrorMessage varchar(100))";
+
+    /*
+    Calling connectionToDatabase method to load the driver and establish the connection.
+    Checking if the Table audit is already exists, if exists we are inserting the values into the table.
+    And closing the connection.
+    If table is not present then calling createNewTableAndInsertValues method so to creat a new table and insert the values.
+     */
+    public void dataBaseStorage(String filePath, String wordSearched, String theResult, int wordCount, String errorMessage) throws SQLException {
```

Comment:
Method name not proper. Java method names should resemble commands

saveDataToDatabase or insertDataToDatabase or storeDataInDatabase etc

Reply:
Updated as per suggestion.